proxy/api/src/keystore.rs

```diff
@@ -16,12 +16,12 @@
 
 use std::convert::Infallible;
 
-use librad::paths::Paths;
 pub use radicle_keystore::pinentry::SecUtf8;
 use radicle_keystore::{
     crypto::{self, Pwhash, SecretBoxError},
     file, Keystore as _, SecretKeyExt,
 };
+use std::path::PathBuf;
```

Comment:
Can you merge this import with the other `std` import above?

Reply:
81781f67277fbe4630ef62761e82ac34e5705523